build/makelib/golang.mk

```diff
@@ -156,6 +156,12 @@ go.test: go.vet go.fmt $(CGO_PREREQS)
 #	@$(GOHOST) test -v -i $(GO_PKG_FLAGS) $(GO_NONSTATIC_FLAGS) $(GO_ALL_PACKAGES)
 	@$(GOHOST) test -cover $(GO_PKG_FLAGS) $(GO_NONSTATIC_FLAGS) $(GO_ALL_PACKAGES)
 
+.PHONY: go.lint
+go.lint: $(GO_TOOLS_DIR)/golint
+	@lint=`$(GO_TOOLS_DIR)/golint $(GO_ALL_PACKAGES)`; \
```

Comment:
@bassam i still need to capture the output to figure out whether there is an error or not. How can you find what error code to set to otherwise?